src/densities.jl

```diff
@@ -79,3 +79,23 @@ be one coefficient matrix per k-Point.
     count = sum(length(basis.ksymops[ik]) for ik in 1:length(basis.kpoints))
     from_fourier(basis, sum(ρaccus) / count)
 end
+
+"""
+Computes the *probability* (not charge) current, ∑ fn Im(ϕn* ∇ϕn)
+"""
+function compute_current(basis::PlaneWaveBasis, ψ::AbstractVector,
+                         occupation::AbstractVector)
+    @assert all(symop -> length(symop) == 1, basis.ksymops) == 1  # TODO lift this
+    current = [zeros(eltype(basis), basis.fft_size) for α = 1:3]
+    for (ik, kpt) in enumerate(basis.kpoints)
+        for (n, ψnk) in enumerate(eachcol(ψ[ik]))
+            ψnk_real = G_to_r(basis, kpt, ψnk)
+            for α = 1:3
+                dαψnk = [im*(basis.model.recip_lattice * (G+kpt.coordinate))[α] for G in G_vectors(kpt)] .* ψnk
```

Comment:
Line too long

Reply:
will be taken care of by the _cart

Reply:
which is not there yet. So than make a TODO to remind yourself to fix it later.

Reply:
I'll do it soon, and I'll just grep for lattice so it won't get forgotten

Reply:
Your name in god's ear as we say in german ;).